controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/filter/AthenzRoleFilter.java

```diff
@@ -46,11 +50,13 @@
 
     private final AthenzFacade athenz;
     private final TenantController tenants;
+    private final ExecutorService executor;
```

Comment:
Should probably be shut down on deconstruction? (Not 100% on whether this applies to filters)

Reply:
That is correct. The filter should inherit `AbstractComponent` and do executor shutdown in `deconstruct()`.